frontend/src/pages/cans/detail/BudgetSummary.jsx

```diff
@@ -48,18 +48,18 @@ const BudgetSummary = () => {
                                     <th>Amount</th>
                                 </tr>
                                 <tr>
-                                    <td>Total FY {cfy[0]?.fiscal_year || "--"} Funding</td>
-                                    <td>{cfy[0]?.total_fiscal_year_funding || "n/a"}</td>
+                                    <td>Total FY {can_fiscal_year[0]?.fiscal_year || "--"} Funding</td>
```

Comment:
I noticed that there's always `[0]` being used.  Is it always an array of size 1?  Can we just return the first item without it being wrapped in an array?

Reply:
I was trying to find away around this, as the Controller is returning an `ListAPIView` so it's always a list. Can't use `RetrieveAPIView` or generic `APIView` (I don't think) as they require the lookup by (pk), unless we wanted to go in and overwrite everything. 

With that said, without knowing more about the full data structure, and whether there's a chance of multiple return values when looked up by `can_id` and `fiscal_year`, I'm not sure yet. The Model does indicate unique on those two values though, so it could be fairly constrained.